src/runtime/components/forms/Form.vue

```diff
@@ -10,7 +10,10 @@ import { useEventBus } from '@vueuse/core'
 import type { ZodSchema } from 'zod'
 import type { ValidationError as JoiError, Schema as JoiSchema } from 'joi'
 import type { ObjectSchema as YupObjectSchema, ValidationError as YupError } from 'yup'
-import type { ObjectSchemaAsync as ValibotObjectSchema } from 'valibot'
+import {
+  type ObjectSchemaAsync as ValibotObjectSchema,
+  safeParseAsync as ValibotParseAsync
```

Comment:
We can't have a direct import to valibot here, because we don't want to include it as a dependency (See this PR: https://github.com/nuxt/ui/pull/617), that's why we used `._parse` in the first place. We need to find another way around.

Reply:
Thanks for answer, do you can see comment @fabien-hiller ? 

https://github.com/nuxt/ui/issues/1782#issuecomment-2106519636

Reply:
In PR #617 valibot is not included in dependencies:

[@nuxthq/ui/v/2.7.0](https://www.npmjs.com/package/@nuxthq/ui/v/2.7.0?activeTab=dependencies)

Now valibot included:

[last version @nuxt/ui](https://www.npmjs.com/package/@nuxt/ui?activeTab=dependencies)

I'm sorry, i try explore in details this question

Reply:
What he meant is as long as no direct import to this dependency is done (except for `import type`) `valibot` is not included in the build. I believe you can bypass this import by using a dynamic import in your function like this:

```suggestion
```

Reply:
> Он имел в виду, что до тех пор, пока не будет выполнен прямой импорт в эту зависимость (за исключением `import type`), `valibot`она не будет включена в сборку. Я считаю, что вы можете обойти этот импорт, используя динамический импорт в своей функции следующим образом:
> 
> См. [https://github.com/nuxt/ui/pull/1783/files#r1599592678 .](https://github.com/nuxt/ui/pull/1783/files#r1599592678)

Oh,  thanks! I check it later and return with answer!